### PR TITLE
Individual Review Thumbnail Component

### PR DESCRIPTION
**Pull request by liu-jeffrey**
## Notion ticket link

[Individual Review Thumbnail Component](https://www.notion.so/uwblueprintexecs/Individual-Review-Thumbnail-Component-73b75f1d3b8f4901a8e565445fcc554b)

## Implementation description
* Made a mock page called mock-search-page with everything
* Four mock thumbnails are on the page: one with normal title length and normal author length, one with long title, one with long author, and one with both long title and author

## Steps to test
1. Go to [http://localhost:3000/test/mock-search-page](http://localhost:3000/mock-search-page)
2. 1st review: 1 book with normal length title + 1 author
3. 2nd review: 2 books with series name "Harry Potter" + multiple authors that overflow
4. 3rd review: 1 book with long title + multiple authors

## What should reviewers focus on?
* matches figma designs https://www.figma.com/file/kEa0TbJ5VKAmI97eGefXs6/Magazine-Review---Subscriber-Experience?node-id=469%3A1748 
* checking if correct props are passed into the thumbnail component. originally we just passed the entire book array.

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have run docker-compose up and my project compiled
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: frontend/src/components/pages/MockSearch/MockSearchPage.tsx [hunk 1]
```diff
@@ -0,0 +1,204 @@
+import { Box } from "@chakra-ui/react";
+import React from "react";
+
+import { Author, Book } from "../../../types/BookTypes";
+import { Review } from "../../../types/ReviewTypes";
+import ReviewThumbnail from "./ReviewThumbnail";
+
+const mockGenericBookData: Book = {
```
Comment (susheelk) [inline frontend/src/components/pages/MockSearch/MockSearchPage.tsx:8]: Could you maybe move this mock data into a separate file? There should be a `mock` folder somewhere

**Review by susheelk — CHANGES_REQUESTED** [review]
LGTM! My only concern is that the thumbnail images don't seem to be responsive:
image
You may find it easier to make them responsive once you've implemented the grid though. (ie your next ticket) Your call!

File: frontend/src/components/pages/MockSearch/MockSearchPage.tsx (hunk 1 above)
Comment (jessiiepeng) [reply frontend/src/components/pages/MockSearch/MockSearchPage.tsx]: Couldn't find the mock folder but created a file called mockReviews.ts and put all of the mock books and reviews in there!

**Review by susheelk — APPROVED** [review]
LGTM!

**Pull request merged**